packages/block-library/src/post-comments/index.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * Server-side rendering of the `core/post-comments` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/post-comments` block on the server.
+ *
+ * @return string Returns the filtered post comments for the current post wrapped inside "p" tags.
+ */
+function render_block_core_post_comments() {
+	$post = gutenberg_get_post_from_context();
```

Comment:
Should we also include the form to add new comments here?

Reply:
Yes, and it should also be editable.

Reply:
`//@TODO` comment for now?

Reply:
I believe these should be two separate blocks (the comment form block is already merged)

Reply:
Oh, right, forgot about that.